go.mod

```diff
@@ -56,7 +56,7 @@ require (
 	github.com/multiformats/go-multiaddr v0.4.1
 	github.com/multiformats/go-multiaddr-dns v0.3.1
 	github.com/onflow/atree v0.1.1
-	github.com/onflow/cadence v0.20.3
+	github.com/onflow/cadence v0.20.4-0.20220124195858-d9317cf57e25
```

Comment:
this has to be updated to a released on, it has been added so tests can run earlier